### PR TITLE
Allows user to select globus origin.

### LINKED ISSUE #3022: Allow user to specify origin location on Globus
1. Allow the user to tell us if their content is on Google Drive, Oak, or Sherlock so that we can send a more precise link that will open the Globus file manager with the origin open on one side and the destination open on the other side. This will make getting to the right place in Globus much easier for users, because it's very challenging to navigate around the Globus interface.
2. Allow the user to tell us if their content is on a different endpoint (by not selecting any of the fixed options) ~~and provide us with the info we need to generate the more precise link to that endpoint. What we need from them may be the UUID for the endpoint, but we need to look into that.~~ 
3. The link in box/step 3 should go to https://docs.google.com/document/d/10b7y3yZCOfyVJ_uP4l7QHbkILQKnfdJi9hj-MpmmdIk/edit (top of the page) and not to a specific header (I'm not sure where that points now). Also, we are not including the "Print instructions" link show in this image.
Updated design: image

4. Make additional changes to the "steps" on top of deposit form as shown here:
https://projects.invisionapp.com/share/YQ132HXFX2GR#/screens/472346509_Globus_Auto_Account

See also #3116, #3021 and #3049 
See designs at https://projects.invisionapp.com/share/YQ132HXFX2GR#/screens/472346509_Globus_Auto_Account

Information from the folks at Globus on how to create these links:
 
- File Manager application links with pre-selected endpoints
— Query parameters supported: destination_id, destination_path, origin_id and origin_path
-- Example https://app.globus.org/file-manager?destination_id=ddb59aef-6d04-11e5-ba46-22000b92c6ec&destination_path=%2F~%2F&origin_id=2fde89c0-6fb4-11eb-8c47-0eb1aa8d4337

Note that we currently have Globus send the link to the shared folder, I think because we didn't know how to figure out what the direct link would be, but they say this is knowable:

origin_id is the endpoint id of source, origin_path is the source path. Those must be available from when you set permission for the user on a specific folder. You can use that to construct the URL.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #3022

# Why was this change made? 🤔

# How was this change tested? 🤨

⚡ ⚠ If this change involves consuming from or writing to another service (or shared file system), ***run [integration test create_object_h2_spec.rb](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test manually in [stage|qa] environment, in addition to specs. ⚡

Unit

# Does your change introduce accessibility violations? 🩺

⚡ ⚠ Please ensure this change does not introduce accessibility violations (at the WCAG A or AA conformance levels); if it does, include a rationale. See the [Infrastructure accessibility guide](https://github.com/sul-dlss/DeveloperPlaybook/blob/main/best-practices/infra-accessibility.md) for more detail. ⚡

### CODE REVIEW

**Comment by justinlittman** [comment]
image

**Review by ndushay — APPROVED** [review]
approving, but does this need PO approval before merge?

**Pull request merged**
